src/app/portlets/dot-edit-content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts

```diff
@@ -175,14 +176,30 @@ describe('DotEditPageToolbarComponent', () => {
         expect(lockSwitch.componentInstance.disabled).toBe(true);
     });
 
-    it('should have page is locked message', () => {
-        component.page.canLock = false;
+    it('should have page is locked by message', () => {
+        component.page.lockedByAnotherUser = true;
         fixture.detectChanges();
 
-        const lockedMessage: DebugElement = de.query(By.css('.edit-page-toolbar__cant-lock-message'));
+        const lockedMessage: DebugElement = de.query(By.css('.edit-page-toolbar__locked-by-message'));
         expect(lockedMessage.nativeElement.textContent).toContain('Page is locked');
```

Comment:
why toContain?, we should use toEquals

Reply:
No, because with the html extra lines the textcontent have a bunch of extra empty spaces.